spec/StringClassSpec.js

```diff
@@ -79,21 +78,20 @@ describe('test isQuestion method', () => {
 
   it('should return false if the string is not a question', () => {
     expect('TBNRTY'.isQuestion()).toBeFalse();
-    expect('tbnrty'.isQuestion()).toBeFalse();
-    expect('245Gy'.isQuestion()).toBeFalse();
+    expect('How are you doing? I\'m great.'.isQuestion()).toBeFalse();
     expect('Thy'.isQuestion()).toBeFalse();
   });
 });
 
 describe('test words function', () => {
   it('should return an array of the string provided', () => {
     expect('tonida'.words()).toEqual(['tonida']);
-    expect('Angel:,Gabriel,is going home'.words())
-    .toEqual(['Angel', 'Gabriel', 'is', 'going', 'home']);
-    expect('Today,tomorrow'.words()).toEqual(['Today', 'tomorrow']);
-    expect(''.words()).toEqual(['']);
-    expect('This is Andela'.words()).toEqual(['This', 'is', 'Andela']);
-    expect(' '.words()).toEqual(['', '']);
+    expect('Angel Gabriel is going home'.words())
+      .toEqual(['Angel', 'Gabriel', 'is', 'going', 'home']);
+    expect('Today, tomorrow'.words()).toEqual(['Today,', 'tomorrow']);
+    expect(''.words()).toEqual([]);
+    expect('This is Andela-TIA'.words()).toEqual(['This', 'is', 'Andela-TIA']);
+    expect('  '.words()).toEqual([]);
```

Comment:
**Code Quality - Critical**
Exclude blanks and spaces in the words array. They are not considered to be words.